pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1737,32 +1739,35 @@ func TestEbpfConntrackerFallback(t *testing.T) {
 	})
 
 	for _, te := range tests {
-		t.Logf("%+v", te)
-		cfg.EnableRuntimeCompiler = te.enableRuntimeCompiler
-		cfg.AllowPrecompiledFallback = te.allowPrecompiledFallback
+		t.Run("", func(t *testing.T) {
```

Comment:
nit: a short descriptive string would be helpful when interpreting test run data (like CI App Visibility)

Reply:
Struggling to come with a short description for the cases :-)